.gitignore

```diff
@@ -1,3 +1,14 @@
 # Ignore (optional) configuration files with user-specific paths:
 initial_cache.cmake
 setup.cfg
+
+# Mac specific files
+*.DS_Store
+
+# IntelliJ IDEA files
+.idea/
+cmake-build-*
+
+# Testing data files
```

Comment:
I don't think we should put `examples/data` in `.gitignore`, as there's already the example images there (and if there are changes to those, we don't want to ignore them). If users have more test data, they should put it somewhere else.

Reply:
Ok. Will change.